Matlab/syntax_test_symbols.matlab

```diff
@@ -8,7 +8,7 @@ classdef BasicClass
     methods
         function obj = BasicClass(val)
 %                      @@@@@@@@@@ definition
-            if nargin == 1
+            if margin == 1
```

Comment:
Hopefully?

Reply:
Nope, `nargin` (number of input arguments) is a builtin variable in Matlab.